### PR TITLE
Task a

### PR DESCRIPTION
**Pull request by nishinotsukasa182**

### CODE REVIEW

**Review by renanwatanabe — COMMENTED** [review]
**sei n heim** 👷

**Review by renanwatanabe — COMMENTED** [review]
teste

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # teste
 teste
-a
\ No newline at end of file
+a
+task1
```
Comment (renanwatanabe) [inline README.md:4]: aa1

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # teste
 teste
-a
\ No newline at end of file
+a
```
Comment (renanwatanabe) [inline README.md:3]: a22

File: README.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+task1
+task2
```
Comment (renanwatanabe) [inline README.md:5]: a33

**Review by renanwatanabe — COMMENTED** [review]
lol

File: README.md (hunk 1 above)
Comment (renanwatanabe) [inline README.md:4]: omg

**Review by renanwatanabe — CHANGES_REQUESTED** [review]
aae

**Review by nishinotsukasa182 — COMMENTED** [review]
aaaa

**Review by nishinotsukasa182 — COMMENTED** [review]
tt

**Review by renanwatanabe — CHANGES_REQUESTED** [review]
yyy

**Review by renanwatanabe — CHANGES_REQUESTED** [review]
lolll

**Review by renanwatanabe — DISMISSED** [review]

**Review by renanwatanabe — DISMISSED** [review]
lgtm

**Review by nishinotsukasa182 — COMMENTED** [review]
renan

**Comment by nishinotsukasa182** [comment]
renan2

**Comment by renanwatanabe** [comment]
aaaa

**Comment by nishinotsukasa182** [comment]
> aaaa

lol

**Pull request closed without merging**